api/workspaces.py

```
@@ -872,8 +873,8 @@ def _get_cli_workspace_tabs(workspace_id: str):
 
             try:
                 messages = traverse_blobs(session["db_path"])
-            except Exception as e:
-                print(f"CLI: could not read session {session_id}: {e}")
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)